core/trino-main/src/main/java/io/trino/sql/gen/JoinCompiler.java

```diff
@@ -1031,14 +1031,15 @@ private BytecodeNode typeEqualsIgnoreNulls(
             BytecodeExpression rightBlock,
             BytecodeExpression rightBlockPosition)
     {
-        MethodHandle equalOperator = typeOperators.getEqualOperator(type, simpleConvention(NULLABLE_RETURN, BLOCK_POSITION, BLOCK_POSITION));
-        BytecodeExpression equalInvocation = invokeDynamic(
+        MethodHandle equalOperator = typeOperators.getEqualOperator(
```

Comment:
Nit: no need to change the wrapping here

Reply:
done